### PR TITLE
fix(node): update all envs to node 4.5

### PR DESCRIPTION
**Pull request by vladikoff**
@jrgm r?

### CODE REVIEW

File: roles/common/defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
 # Note: `nave` requires a dot in the version, so for nodejs 4.x, you must
 # specify 4.0 or 4.1 or 4.2 or 4.3 (or what ever minor version in the future)
-node_version: "0.10"
+node_version: "4.5"
```
Comment (pdehaan) [inline roles/common/defaults/main.yml]: Node 4.5.0 'Argon' includes npm 2.15.9 (per https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V4.md#4.5.0)

Diff: https://github.com/npm/npm/compare/v2.14.18...v2.15.9

Not sure if you also want to bump the `npm_version` below or not.

Comment (vladikoff) [inline roles/common/defaults/main.yml]: Thanks @pdehaan !

**Comment by jrgm** [comment]
LGTM. Removing the `npm_version` is good. Current nodejs 0.10 and up ship with a 2.x or greater npm, and we can just take whatever npm comes with nodejs (which in 6, may be npm@3, iirc).

**Comment by shane-tomlinson** [comment]
🔥! This is awesome, great work guys!

**Comment by pdehaan** [comment]
Related, not related: https://nodejs.org/en/blog/vulnerability/september-2016-security-releases/

> We intend to make releases available on or soon after the evening of **Tuesday, the 27th of September, 2016, UTC** (midday US time).
> ...
> These are the expected version numbers for the releases:
> - Node.js v6.7.0 (Current)
> - Node.js v4.6.0 (LTS "Argon")

**Comment by jrgm** [comment]
Thanks pdehaan. Yeah, was planning on this bump in stage/prod next week when it's available, but we'll also need to update in fxa-dev. 

https://github.com/mozilla/fxa-dev/issues/273

**Pull request merged**